src/gl.rs

```diff
@@ -58,7 +58,13 @@ pub fn delete_frame_buffers(frame_buffers: &[GLuint]) {
 pub fn read_pixels(x: GLint, y: GLint, width: GLsizei, height: GLsizei, format: GLenum, pixel_type: GLenum) -> Vec<u8> {
     let colors = match format {
         ffi::RGB => 3,
+        ffi::BGR => 3,
+
         ffi::RGBA => 4,
+        ffi::BGRA => 4,
```

Comment:
I think BGRA is an extension on android opengl es - so it may cause compile errors when we update gleam. But we can deal with that when it occurs.